lib/docker-toolbelt.coffee

```diff
@@ -311,7 +311,11 @@ Docker::getRegistryAndName = Promise.method (image) ->
 # Example: { registry: "registry.resinstaging.io", imageName: "resin/rpi", tagName: "1234"}
 #		=> registry.resinstaging.io/resin/rpi:1234
 Docker::compileRegistryAndName = Promise.method ({ registry, imageName, tagName }) ->
-	registry += '/' if registry? and registry isnt ''
+	if registry? and registry isnt ''
+		registry += '/'
+	else
+		# For the undefined case
+		registry = ''
```

Comment:
I knew there would be a coffeescript shorthand for it, I just couldn't remember what it was! Nice, cheers